src/Visitor.php

```diff
@@ -1,18 +1,19 @@
 <?php
 
-namespace Core;
+namespace GroupLife\Core;
+
```

Comment:
Hmmm, I already saw this change in different pull request, something is wrong here.

Reply:
still have problems with what to take and where to get it from

Reply:
If you did come change in different PR, is it not necessary to repeat it. The change will propagate eventually when PR will be accepted.